tests/templates/test_embeddings.py

```diff
@@ -455,22 +427,6 @@ def circuit(f=None):
         with pytest.raises(ValueError, match="'features' must be of shape"):
             circuit(f=features)
 
-    @pytest.mark.parametrize('pattern', [qml.RZ,
-                                         [qml.RZ, qml.RZ],
-                                         [[qml.RZ, qml.RZ], [qml.RZ, qml.RZ]]])
-    def test_exception_wrong_type_pattern(self, pattern):
```

Comment:
How come this test is removed?

Reply:
The Wires class tests for the case that wires have a funky format, so it's not needed here any more...

Reply:
Hmm, it seems that the wires are defined here in a valid way (`range(3)`) and the type of pattern is being tested for cases that are considered invalid. How can the `Wires` class help out with checking that?

Reply:
The Wires class will turn ``[qml.RZ, qml.RZ]`` into ``Wires([qml.RZ, qml.RZ])`` and then find that the pattern contains irregular inputs. 

Do you prefer this test to remain here (since it addresses the pattern)? I could just adapt it...